apps/api/src/app/controllers/__tests__/fetchManyViewModels.e2e.spec.ts

```diff
@@ -38,10 +45,16 @@ describe('When fetching multiple resources', () => {
     );
 
     beforeAll(async () => {
-        ({ app, arangoConnectionProvider, testRepositoryProvider } = await setUpIntegrationTest({
-            ARANGO_DB_NAME: testDatabaseName,
-            BASE_DIGITAL_ASSET_URL: 'https://www.mysound.org/downloads/',
-        }));
+        ({ app, arangoConnectionProvider, commandHandlerService, testRepositoryProvider } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: testDatabaseName,
+                BASE_DIGITAL_ASSET_URL: 'https://www.mysound.org/downloads/',
+            }));
+
+        commandHandlerService.registerHandler(
```

Comment:
Put a TODO here to use the actual app module and dynamically register all commands as a result.

Reply:
done